### DISCUSSION TITLE
Bounding boxes wrong for rotated models

### DISCUSSION THREAD

**Discussion by holycowbatmann** [raysan5/raylib#3844] [General]
Im using getModelBoundingBox with drawBoundingBox with the Golang binding and as you can see the red outline for the bounding box is completly wrong if you rotate the model... it works right with no rotation. Am i missing something.

![bboxes](https://github.com/raysan5/raylib/assets/114120704/1c4befec-a9fe-40ba-b895-bb11807a17e5)

**Comment by raysan5** [comment]
@holycowbatmann I'm afraid BoundingBoxes do not support rotation, they are axis-aligned, it's up to the users to find a better alternative, for example using a 3d physics library.
